front/src/modules/timesStops/types.ts

```diff
@@ -1,5 +1,12 @@
 import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
 
-export type PathWaypointColumn = SuggestedOP & {
+export type PathWaypointRow = SuggestedOP & {
   isMarginValid: boolean;
 };
+
+export enum TableType {
+  Input = 'Input',
+  Output = 'Output',
+}
+
+export type PathPointOpPointCorrespondance = { correspondingOpPointIndex: number };
```

Comment:
Use PathStep instead of pathPoint. Also maybe opPoint is a bit redundant, op (for operation point) is not enough ? (only a suggestion)
```suggestion
export type PathStepOpPointCorrespondance = { correspondingOpPointIndex: number };
```

Reply:
done. I guess `opPoint` is clearer